functions/airtableTriager.js

```diff
@@ -0,0 +1,16 @@
+const functions = require('firebase-functions');
+
+const { getChangedIntakeTickets } = require('./airtable');
+
+// Runs every minute
+exports.poll = functions.pubsub.schedule('* * * * *').onRun(async () => {
```

Comment:
naming wise i think the base namespace should be `poll` or `poller` and then this individual function would be `airtableTickets` or something